airbyte-integrations/connectors/source-jira/source_jira/schemas/labels.json

```diff
@@ -1,33 +1,24 @@
 {
-  "$schema": "http://json-schema.org/schema#",
+  "type": ["object", "null"],
```

Comment:
Should there be a
`"$schema": "http://json-schema.org/draft-07/schema#",` 
too?

Reply:
autogenerated schema always have this entry `$schema`. The swagger big hairy json is https://developer.atlassian.com/cloud/jira/platform/swagger.v3.json . Accordingly to it, the labels are https://developer.atlassian.com/cloud/jira/platform/rest/v3/api-group-labels/#api-group-labels PageBeanString. In this json document it is described here `root['components']['schemas']['PageBeanString']` and it just does not contain $schema entry. Schemas were restored to the state before https://github.com/airbytehq/airbyte/pull/4947 and it seems that schemas, as well swagger one's just does not contain this entry. We can modify existing jsons and set it, BUT the previous version and swagger do not have it, and we use swagger.